src/main/java/com/github/manolo8/darkbot/utils/StartupParams.java

```diff
@@ -0,0 +1,118 @@
+package com.github.manolo8.darkbot.utils;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.Properties;
+
+public class StartupParams {
+    private static final String COMMAND_PREFIX = "-";
+
+    /**
+     * Command-line argument for auto-login
+     * requires a path to a properties file containing a username and either a password or a master-password.
+     * Example usage: {@code -login C:\Users\Owner\startup.properties}
+     *
+     * @see #startupPropertiesPath
+     */
+    private static final String LOGIN_COMMAND = COMMAND_PREFIX + "login";
+    private boolean autoLogin = false;
+    /**
+     * Path to properties file containing 3 keys: username, password, master-password
+     * username is required to be defined and you can choose to define either password or master-password
+     * If you have an empty master-password you can define that field to be empty
+     * leave undefined fields empty
+     */
+    private String startupPropertiesPath;
+
+    /**
+     * Command-line argument for auto-start, has no parameters.
+     */
+    private static final String START_COMMAND = COMMAND_PREFIX + "start";
+    private boolean autoStart = false;
+
+    private String[] args;
+    private Properties properties;
+
+    public StartupParams(String[] args) {
+        this.args = args;
+        parse(args);
+        System.out.println(this);
+    }
+
+    private void parse(String[] args) {
+        for (int i = 0; i < args.length; i++) {
+            switch (args[i]) {
+                case LOGIN_COMMAND:
+                    if (i + 1 >= args.length) {
+                        System.err.println("Missing arguments for auto-login, usage: -login path/to/startup.properties");
+                        System.exit(-1);
+                    }
+                    startupPropertiesPath = args[++i];
```

Comment:
```suggestion
                    properties = loadLoginProperties(args[++i]);
```
Just load the properties here, make parsing be able to throw that IOException